### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Facility Locator mobile map iteration

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#104185]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I'm sorry for the delayed written feedback! I dropped the ball this week and this fell off my radar. Please let me know if you have any questions or if my slowness impacted your team at all.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** I think you can trim down the content for the TTY numbers for each result, removing the label explaining that they're for the relay service. Love including TTY for them, but TTY users know what it is and non-TTY users are fine ignoring. Note that the [content style guide](https://design.va.gov/content-style-guide/dates-and-numbers#formatting) has TTY numbers following the main number as a parenthetical. I don't know if that fits these results perfectly but you may be able to work with the content folks at CAIA to come up with a cleaner solution.
- [ ] **Should:** Echoing Erin's comments (#103844) on the extra motion on the map when toggling between views --- if possible, try to avoid this behavior. I don't think it quite reaches the standard of excessive motion that could trigger motion sickness or a vestibular condition, but it's definitely something to be careful about.
- [ ] **Engineering note:** Related to that, I'd encourage you to see if map animations can be removed entirely for users with the [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) flag. That may be something already bundled in with the map library you're using.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by jilladams** [comment]
The `should` re: TTY content will get addressed by https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20395

@Agile6MSkinner for these 2, we'll need tickets to investigate / track: 
1. Should: Echoing Erin's comments (https://github.com/department-of-veterans-affairs/va.gov-team/issues/103844) on the extra motion on the map when toggling between views --- if possible, try to avoid this behavior. I don't think it quite reaches the standard of excessive motion that could trigger motion sickness or a vestibular condition, but it's definitely something to be careful about.
2. Engineering note: Related to that, I'd encourage you to see if map animations can be removed entirely for users with the [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) flag. That may be something already bundled in with the map library you're using.
